src/lib/components/modal/ModalActions.svelte

```diff
@@ -0,0 +1,3 @@
+<div class="flex justify-between mt-5">
+  <slot />
+</div>
```

Comment:
I think we can also add a ModalFooter section and inside that we can put ModalActions if needed.

Reply:
Hmm, after a quick check of other component libraries, it looks like we should choose _either_ `ModalActions` or `ModalFooter`.

Vercel uses `ModalActions`: https://vercel.com/design/modal
MaterialUI uses `DialogActions`: https://mui.com/material-ui/react-dialog/
ChakraUI uses `ModalFooter`: https://chakra-ui.com/docs/components/modal

Since Modals by definition require user interaction (source: https://www.nngroup.com/articles/modal-nonmodal-dialog/), okay if we keep the "Action" terminology?

Reply:
Yes sure, that works!